src/lib/auth/auth-cli.rb

```diff
@@ -0,0 +1,23 @@
+Yast.import "CommandLine"
+
+module Auth
+  class CLI
+    extend Yast::I18n
+
+    def self.run(cmd="auth")
+      textdomain "auth"
+
+      msg = format(
+        _("Command line is not supported. Use '%s' directly."),
```

Comment:
Same here. I am not sure if ldapkrb and auth-client is CLI tools. You can look for example at storage where is also no suitable CLI tool: https://github.com/yast/yast-storage-ng/pull/1170/files#r527675569

Reply:
For sure I didn't see any existing use of 'CommandLine', whether ldapkrb, auth-client, krb-server or ldap-server should really provide a cli interface is not a question personally I can answer, I am only trying to solve the immediate problem of the bash completion here.

Not really familiar with this stuff so looking at the yast-storage example I am guessing that it is the textdomain & msg/format use here is what you are unhappy about :-)

I'll push a new version with that stuff removed to this & auth-server